src/main/com/aca/homework/week8/random/generator/Result.java

```diff
@@ -0,0 +1,23 @@
+package com.aca.homework.week8.random.generator;
+
+public class Result {
+    private final String text;
+    private final double duration;
+
+    private Result(String text, double duration) {
+        this.text = text;
+        this.duration = duration;
+    }
+
+    public static Result resultOf(String text, double duration) {
```

Comment:
no need to create a static factory. the static factory is not better than the constructor in this case.